scripts/test

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${RF_API_SPEC_DEBUG}" ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n \
+         "Usage: $(basename "$0")
+Run linters and tests."
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]
+then
+    if [ "${1:-}" = "--help" ]
+    then
+        usage
+    else
+        # Validate spec
+        docker-compose \
+            run --rm app \
+            yarn run swagger-cli validate spec.yml
```

Comment:
I think that this approach to executing the `swagger-cli` is fine, but it may be worth considering adding this bit to the `package.json` file under a `test` target to align with ecosystem conventions.

```javascript
  "scripts": {
    "test": "..."
  }
```